src/v2/Components/__stories__/Gene.story.tsx

```diff
@@ -13,6 +13,7 @@ storiesOf("Components/Pages/Gene/Contents", module)
             filters={{}}
             geneID="minimalism"
             mode="artists"
+            // eslint-disable-next-line no-console
```

Comment:
I see a lot of ignores from stories here. I think it'd be safe to configure an [override](https://eslint.org/docs/user-guide/configuring#specifying-processor) in the `.eslintrc.js` file that would allow us to disable the log rules for story files.

It'd look something like:
```json
{
  "overrides": [{
    "files": ["src/v2/**/*.story.tsx"],
   "rules": {
     "no-console": "off"
   }
  }]
}
```